backend/github.js

```diff
@@ -53,3 +53,26 @@ exports.retrieveOrganizations = function (accessToken, callback) {
     callback(null, result);
   });
 };
+
+/**
+ * Delete a Repository WebHook
+ * @param name: name of the Repository
+ * @param hook: Hook Id
+ * @param accessToken: Access Token of the user
+ * @param callback
+ */
+exports.deleteHook = function (name, hook, accessToken, callback) {
```

Comment:
There is more code written using callbacks than using Promises, updating the structure of Yaydoc would require a lot of time. Using callbacks is more of a readability problem than usages problem.
The code changes could be done at later stages, if required.